src/components/profile/loansAndReservationsPage/Page.js

```diff
@@ -49,14 +72,57 @@ export const dataReducer = (dataType, data) => {
   }
 };
 
+const sortLoans = (loans) => {
```

Comment:
Evt. kan de her funktioner være i et factory, men som minimum skal de være i en `.utils.js`-fil.

Det er hvis jeg forstår ret, det samme på tværs af alle visninger af order, så det kunne argumentere for en factory